frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/create-local-volume-set/body.tsx

```diff
@@ -0,0 +1,321 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  FormGroup,
+  TextInput,
+  Radio,
+  ExpandableSection,
+  TextInputTypes,
+  Text,
+  TextVariants,
+  Tooltip,
+} from '@patternfly/react-core';
+import { ListPage } from '@console/internal/components/factory';
+import { Dropdown } from '@console/internal/components/utils';
+import { NodeModel } from '@console/internal/models';
+import { NodeKind } from '@console/internal/module/k8s';
+import { getName, MultiSelectDropdown } from '@console/shared';
+import {
+  deviceTypeDropdownItems,
+  diskSizeUnitOptions,
+  diskTypeDropdownItems,
+} from '@console/local-storage-operator-plugin/src/constants';
+import { NodesTable } from '@console/local-storage-operator-plugin/src/components/tables/nodes-table';
+import { diskModeDropdownItems, NO_PROVISIONER } from '../../../../constants';
+import { WizardDispatch, WizardState } from '../../reducer';
+
+import './body.scss';
+
+export const LocalVolumeSetBody: React.FC<LocalVolumeSetBodyProps> = ({
+  dispatch,
+  state,
+  storageClassName,
+  taintsFilter,
+  diskModeOptions = diskModeDropdownItems,
+  allNodesHelpTxt,
+  lvsNameHelpTxt,
+  deviceTypeOptions = deviceTypeDropdownItems,
+}) => {
+  const { t } = useTranslation();
+  const formHandler = React.useCallback(
+    (field, value) =>
+      dispatch({ type: 'wizard/setCreateLocalVolumeSet', payload: { field, value } }),
+    [dispatch],
+  );
+
+  const diskDropdownOptions = diskTypeDropdownItems(t);
+
+  const INTEGER_MAX_REGEX = /^\+?([1-9]\d*)$/;
+  const INTEGER_MIN_REGEX = /^\+?([0-9]\d*)$/;
+  const [activeMinDiskSize, setMinActiveState] = React.useState(false);
+  const [activeMaxDiskSize, setMaxActiveState] = React.useState(false);
+  const validMinDiskSize = INTEGER_MIN_REGEX.test(state.minDiskSize || '1');
```

Comment:
```suggestion
  const validMinDiskSize = INTEGER_MIN_REGEX.test(state.minDiskSize || '0');
```
by above logic, i think it can zero too ?

Reply:
lso recommends to use min disk size 1

Reply:
above regex seems to accept 0 too if the state is that, maybe we can just use the integer max regex